docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,52 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main feature of the Metadata API is to extract metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.
+
+
+## Prerequisites
+
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html), and [tv](../../api/latest/device_api/tv/tizen/metadata.html) applications), the application has to request storage privileges by adding the following privileges to the `config.xml` file:
+
```

Comment:
> @mijin85cho I presume that the page is new one, two links that are not working will work afterwards

@nawab-samsung As I guided you before with https://github.com/Samsung/tizen-docs/pull/1076#discussion_r409391314,
you can change the "latest" in the address bar, to actual version (6.0 in this case) when you check on "View file" tab.

Please don't be confused that it is not asking you to guide the author to change the latest to 6.0,
as it is just a way for the reviewers to check the validation of the links to API.

Or, as pkosko guided, or guided on https://github.com/Samsung/tizen-docs/pull/1076#discussion_r409391314,
you can check the links on the staging URL.

Reply:
@mijin85cho thanks, I do remember, but for me staging was down after multiple tries, when I tried to check so to ensure status of links, just asked @pkosko to verify the link but not change.